src/Backend/Modules/Slideshows/Actions/Add.php

```diff
@@ -46,6 +50,7 @@ private function handleForm()
                 // build item
                 $item['language'] = Language::getWorkingLanguage();
                 $item['title'] = $txtTitle->getValue();
+                $item['template'] = $template->getValue();
```

Comment:
you should only show/add this dropdown if there are more than 1 possible template, just like we do in the Content Blocks module else you should fallback to the default